src/Microsoft.Health.Fhir.Core/Configs/ExportJobConfiguration.cs

```diff
@@ -45,5 +46,11 @@ public class ExportJobConfiguration
         /// Number of pages to be iterated before committing the export progress.
         /// </summary>
         public uint NumberOfPagesPerCommit { get; set; } = 10;
+
+        /// <summary>
+        /// Formats for export jobs.
+        /// </summary>
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Usage", "CA2227:Collection properties should be read only", Justification = "This is a configuration class")]
+        public IList<ExportJobFormatConfiguration> ExportJobFormats { get; set; }
```

Comment:
>ExportJobFormats [](start = 51, length = 16)

nit: This is already part of ExportJobConfiguration, so putting ExportJob in front of Formats sound a bit excessive for me.